pkg/csi/service/wcp/controller.go

```diff
@@ -1094,8 +1085,22 @@ func (c *controller) ControllerUnpublishVolume(ctx context.Context, req *csi.Con
 					})
 					if err != nil {
 						if isStillAttached {
-							log.Errorf("volume %q is still attached to node %q and podVM %q with moId %q", req.VolumeId,
-								req.NodeId, podVM.Reference().String(), podVM.Reference().Value)
+							volumeId := []string{req.VolumeId}
```

Comment:
Can we add some comments indicating why we're introducing this code block?

Reply:
Done